app/src/main/java/ru/yandex/practicum/contacts/presentation/sort/SortTypeUI.java

```diff
@@ -2,9 +2,11 @@
 
 import androidx.annotation.NonNull;
 
+import ru.yandex.practicum.contacts.presentation.base.ListDiffInterface;
 import ru.yandex.practicum.contacts.presentation.sort.model.SortType;
 
-public class SortTypeUI {
+/** @noinspection rawtypes*/
+public class SortTypeUI implements ListDiffInterface {
```

Comment:
И в этом классе тоже нужно поправить сигнатуру класса и метод `theSameAs`:
```suggestion
public class SortTypeUI implements ListDiffInterface<SortTypeUI> {
```